scripts/generated/DummyDiamondImplementation.sol

```diff
@@ -0,0 +1,201 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
we can maybe gitignore this one?

Reply:
No better have it other wise you necessarily need to fork the other repo generating this dummy file